prime-angular-frontend/src/app/modules/enrolment/shared/components/college-certification-form/college-certification-form.component.html

```diff
@@ -36,7 +36,9 @@
                    placeholder="Licence Number"
                    formControlName="licenseNumber">
             <mat-error *ngIf="form.get('licenseNumber').hasError('required')">Required</mat-error>
-            <mat-error *ngIf="form.get('licenseNumber').hasError('length')">Must be a 5 digit number</mat-error>
+            <mat-error *ngIf="form.get('licenseNumber').hasError('alphanumeric')">Must contain only be alphanumeric
+              characters
+            </mat-error>
           </mat-form-field>
```

Comment:
Looks like there's a typo in the mat-error.  It contains "be" from previous version